### ISSUE TITLE
stable build doesn't seem to point to version tags

### ISSUE THREAD

**Issue by jagerber48** [readthedocs/readthedocs.org#10576]
## Details

* Read the Docs project URL: [https://readthedocs.org/projects/sciform/](https://readthedocs.org/projects/sciform/)

## Expected Result

I expect the `stable` build on read the docs to be linked to the most recent semver tag Le.g. `0.22.2`) that appears in the github repo that the RTD project is linked to. I expect the RTD to update it's list of tags either when new tags are pushed or deleted from the repo or on build events.

## Actual Result

RTD was following my expected behavior up until version `0.21.0` on my repo. However, when I added tags for `0.22.0`, `0.22.1`, and `0.22.2` to my repo in the past 24 hours I did not see the expected behavior. These tags appeared in my RTD version list and the `latest` build correctly tracked to the most recent version of the `main` branch of the repo as configured. However, the `stable` branch was *stuck* always building on the commit pointing to version `0.21.0`. 

I tried a number of things 

- Repeated user-triggered builds of `stable` and `latest`
- Deactivating and re-activating both the `latest` and `stable` versions on RTD. 
- I tried removing and re-adding tags to the git repo.

Nothing I did would cause `stable` to manually point to the latest semver tag. I was able to get RTD `stable` to point to whichever commit I wanted by adding a `stable` tag into the git repo. When I did this the RTD `stable` pointed to the new `stable`-tagged commit (as indicated in the "versions" tab for the project) within seconds. However, upon removing the `stable` tag, RTD was still stuck pointing to whatever commit I had put the `stable` tag on and would not update to the latest semver tag. Even if the tag was added or removed or the RTD `stable` was de-activated and re-activated.

Right the both `stable` and `latest` correctly point to the `0.22.2`-tagged commit. But `stable` is only pointing there because I created and deleted a `stable` tag on that commit. I'm concerned that when I bump to `0.22.3` or `0.23.0` that the RTD `stable` build automation will fail again and continue to point to this same commit.

Possibly relatedly: I found old issues from 2016-2018 or so with similar issues but it seems like they were solved by updates to RTD. Nonetheless, those issues sometimes suggested deleting and re-creating the project. I reluctantly tried this but the deletion failed and was met with a "try again later" error.

I've tried everything I can think of. It seems the issue may be something with RTD or my RTD project. But it's very possible it could be an issue with how I have my project or github repo configured.

**Comment by stsewd** [comment]
Hi, looks like an error from our side, we deleted an unused table, but not the app from django, this is causing errors in our sync versions code, we are working on a fix.

**Comment by stsewd** [comment]
We have deployed a fix, I think re-building any version should update your stable version to point to the correct one. Let us know if that works.

**Comment by aanm** [comment]
I can confirm, the issue is now fixed. Thank you!

**Comment by jagerber48** [comment]
I can also confirm the issue is fixed for me! I don't know the exact logic for when things are supposed to be triggered, but I deleted my most recent tag and then when I requested to build `stable`, RTD was able to resolve the previous version tag and build that version. Then when I re-pushed the most recent version tag to github, RTD was able to automatically detect this and build `stable` to this correct tag.

Thank you very much for your very speedy response to this issue!
